doc/source/conf.py

```diff
@@ -77,13 +79,27 @@
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
+breathe_projects = {"arrow_cpp": "../../cpp/apidoc/xml"}
+breathe_default_project = "arrow_cpp"
+
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
-# source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
 
-autosummary_generate = glob.glob("*.rst")
+class CMWithLicenseParser(CommonMarkParser):
+
+    def parse(self, inputstring, document):
+        # Strip the license header from the MarkDown file. Sadly recommmark cannot handle this.
```

Comment:
i'm -1 on including MarkDown docs in the Sphinx source tree. We should convert those documents to reST format. It's a bit of work, but it will be much nicer afterwards.

Reply:
I don't think anyone has ever proposed converting the format docs to another markup language. In principle I don't see an issue with using reST because people will mostly be consuming the documents via the generated webpage. These aren't documents that are going to be frequently edited, anyway

Reply:
Ok, then I'll proceed and convert them to reST.